client/my-sites/navigation/index.jsx

```diff
@@ -18,6 +20,15 @@ class MySitesNavigation extends React.Component {
 		event.stopPropagation();
 	};
 
+	renderSidebar() {
+		if ( 'jetpack-cloud' === config( 'sidebar' ) ) {
+			return (
+				<JetpackCloudSidebar path={ this.props.path } siteBasePath={ this.props.siteBasePath } />
```

Comment:
Including both `JetpackCloudSidebar` and `Sidebar` here is having the side effect of loading both of their css rules. Because we use a .12 alpha for Jetpack Cloud it is dramatically changing the selected color in themes with colored sidebars:
**Production**
<img width="500" alt="Screen Shot 2020-03-02 at 4 57 50 PM" src="https://user-images.githubusercontent.com/2810519/75732365-251a5d00-5ca7-11ea-9d2d-50cec1d99ac3.png">
**Branch**
<img width="500" alt="Screen Shot 2020-03-02 at 4 57 44 PM" src="https://user-images.githubusercontent.com/2810519/75732370-28154d80-5ca7-11ea-8221-5a4b4f082d46.png">

Reply:
Nice catch. I wonder if the same thing applies to the masterbar. 
Will look into fixing this.